alibi_detect/utils/pytorch/distance.py

```diff
@@ -93,7 +115,43 @@ def batch_compute_kernel_matrix(
     return k_mat
 
 
-def mmd2_from_kernel_matrix(kernel_mat: torch.Tensor, m: int, permute: bool = False,
+def linear_mmd2(x: torch.Tensor,
```

Comment:
Nitpick but probs worth keeping indentation within function definitions consistent with all of the other functions.

Reply:
Fixed, should be consistent all across now.